### DISCUSSION TITLE
Client Size Preset Selection

### DISCUSSION THREAD

**Discussion by Nathun56** [runelite/runelite#16127] [Ideas]
Would love there to be a plugin for custom client size selection. Would be great if there was a sidebar section with addable buttons to different client sizes; which would be great for someone like me who is always adjusting the size of my runelite client. 

Many thanks :)

**Comment by antero111** [comment]
You can do this with Plugin Presets. See [Client-size-presets](https://github.com/antero111/plugin-presets/wiki/Client-size-presets) in the Plugin Presets wiki for a how-to.

**Reply by Nathun56** [reply]
Hey, thank you for replying. the Plugin Presets plugin is not working for me at the moment, could there be any conflicting plugins that are stopping the side bar from showing up?

Thank you

**Reply by antero111** [reply]
Plugin Presets should not be the cause for the RuneLite sidebar not showing. If its the Plugin Presets tab then please create an issue for it in the Plugin Presets repo. Afaik no plugin conflicts with Plugin Presets.

I'd suggest that you check the RuneLite Discord for support.
